service/src/main/java/uk/gov/hmcts/reform/fpl/model/Child.java

```diff
@@ -7,12 +7,13 @@
 import lombok.Data;
 
 import java.util.Date;
+import javax.validation.constraints.NotBlank;
 
 @Data
 @Builder
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class Child {
-
+    @NotBlank(message = "Tell us the names of all children in the case")
```

Comment:
Again I am really not sure about the wording of these error messages, have they all been checked with Mia and Eilis. Tell us the names to me really doesnt sound like an official error message to me, I will not comment on any more of these but I would double check some of them because to me they do not read well.